packages/client/src/components/Textarea/Textarea.tsx

```diff
@@ -0,0 +1,36 @@
+import { ChangeEvent, InputHTMLAttributes, useState } from 'react';
+import cn from 'classnames';
+import styles from './textarea.module.scss';
+
+export interface TextareaProps
+  extends InputHTMLAttributes<HTMLTextAreaElement> {
+  allowedCharactersNumber: number;
+  errorMessage?: string;
+}
+
+export const Textarea = (props: TextareaProps) => {
+  const { errorMessage, allowedCharactersNumber, ...otherProps } = props;
```

Comment:
Может будем сразу деструктурировать просы?
```suggestion
  export const Textarea = ({ errorMessage, allowedCharactersNumber, ...otherProps }: TextareaProps): JSX.Element => {
```

Reply:
думаю пока так оставить, потом если решм переделать то одним коммитом и ПР вольем из какой-нибудь ветки fix, так как все равно править в том что уже залито нужно, уточню в пачке